roxie/ccd/ccdstate.cpp

```diff
@@ -176,14 +176,15 @@ class DelayedReleaserThread : public Thread
         unsigned nextTimeout = INFINITE;
         while (!closing || queue.length())
         {
-            sem.wait(nextTimeout);
+            if (!closing)
```

Comment:
Not sure about this - it will also prevent a roxie node from persisting long enough to complete active agent queries when it is closed by an autoscale operation, I think